orderbook/src/bad_token_cache.rs

```diff
@@ -0,0 +1,53 @@
+use crate::bad_token::{BadTokenDetector, TokenQuality};
+use anyhow::Result;
+use primitive_types::H160;
+use std::{collections::HashMap, sync::Mutex};
+
+pub struct BadTokenCache {
+    detector: BadTokenDetector,
+    // Std mutex is fine because we don't hold lock across await.
+    cache: Mutex<HashMap<H160, TokenQuality>>,
```

Comment:
A std Mutex is fine here, but why not use a `tokio::sync::Mutex` anyway? I would imagine it performs much better.